lib/search.js

```diff
@@ -96,12 +88,13 @@ module.exports = function (client, value, from, size, effect, weights) {
                     },
                     script_score: {
                         lang: 'groovy',
-                        script: 'pow(' +
+                        script: '(doc["name.raw"].value.equals(name)) ? 1 : _score * pow(' +
                         'doc["score.detail.popularity"].value * weightPopularity + ' +
                         'doc["score.detail.quality"].value * weightQuality + ' +
                         'doc["score.detail.maintenance"].value * weightMaintenance, ' +
                         'effect)',
                         params: {
+                            name: value.toLowerCase(),
```

Comment:
As we discussed privately @atduarte we should do this at the head of this function. Can you change that so that it goes into this PR?